frontend/src/test/scala/bloop/HydraCompileSpec.scala

```diff
@@ -56,10 +53,10 @@ object HydraCompileSpec extends BaseCompileSpec {
     }
   }
 
-  private lazy val hydraLicenseExists: Boolean = {
```

Comment:
We already ignore hydra tests if the license is not available from what I remember? Might be ok to remove this part.

Reply:
A few lines below, in https://github.com/scalacenter/bloop/pull/1619, you commented out an if which was using `hydraLicenseExists`. I think that this part should be removed only if  `HydraCompileSpec` will be removed.